### DISCUSSION TITLE
how do I get white text on black background in EPUBs WITHOUT inverting the colors of images???

### DISCUSSION THREAD

**Discussion by cawphee-black** [sumatrapdfreader/sumatrapdf#4123] [Q&A]
What is going on? In some earlier versions there was a EBookUI set of settings that worked great for EPUBs but that seems to be gone now.
Now if I set the textcolor to #eeeeee and background color to #000000; the result is white text on a black background BUT the images are are inverted in color!!

Why does changing what should be text only settings do this?
And  if I change InvertColors to true, it changes the both the images and the text so I get White Background and Black Text.

```
FixedPageUI [
	TextColor = #eeeeee
	BackgroundColor = #000000
	SelectionColor = #f5fc0c
	WindowMargin = 2 4 2 4
	PageSpacing = 4 4
	InvertColors = false
	HideScrollbars = false
]
```
this results in black background and white text BUT inverted images!! WHY????

```
FixedPageUI [
	TextColor = #eeeeee
	BackgroundColor = #000000
	SelectionColor = #f5fc0c
	WindowMargin = 2 4 2 4
	PageSpacing = 4 4
	InvertColors = true
	HideScrollbars = false
]
```
this results in WHITE BACKGROUND AND BLACK TEXT but NORMAL IMAGES... wtf?

How do I fix this or do we need a new version of sumatra to do that?

These are the ebook settings I used to be able to use
```
EbookUI [
	FontName = Georgia
	FontSize = 16
	TextColor = #eeeeee
	BackgroundColor = #000000
	UseFixedPageUI = false
]
```
and did the job right... but the settings file with this in it get that section removed by the v3.5.2 of sumatra

**Comment by GitHubRulesOK** [comment]
The ePub version 1 (Legacy) ebook and text formats viewer, were replaced with the same base engine as used for PDF (MuPDF rendering) so should be nearer ePub level 2 . this means its the same MuPDF "filter" (one range of colours between two stops).

Thus if you set colour band from top to bottom  all colours can ONLY be reversed like an Xray or camera negative.

**Comment by kjk** [comment]
In [pre-release](https://www.sumatrapdfreader.org/prerelease) there are epub customization options. See https://www.sumatrapdfreader.org/docs/Customize-eBook-UI
